From randy.dunlap@oracle.com  Fri Sep 11 15:56:39 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 7 Sep 2009 17:08:39 -0700
Subject: USB: gadget: ether needs to select CRC32
To: "Stephen Rothwell . David Brownell <dbrownell@users.sourceforge.net>" <sfr@canb.auug.org.au>, gregkh <greg@kroah.com>
Message-ID: <20090907170839.50ddeef7.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix build error, ether uses/needs to select CRC32 config symbol:

ether.c:(.text+0x271480): undefined reference to `crc32_le'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/gadget/Kconfig
+++ b/drivers/usb/gadget/Kconfig
@@ -627,6 +627,7 @@ config USB_AUDIO
 config USB_ETH
 	tristate "Ethernet Gadget (with CDC Ethernet support)"
 	depends on NET
+	select CRC32
 	help
 	  This driver implements Ethernet style communication, in one of
 	  several ways:
